plugins/gcloud/lib/samson_gcloud/image_scanner.rb

```diff
@@ -49,7 +47,8 @@ def status(id)
       # NOTE: If we could guarantee ordering we could make a single request for
       # "(kind=\"DISCOVERY\" OR kind=\"PACKAGE_VULNERABILITY\")"
       def uncached_scan(image)
-        return ERROR unless image.include? SamsonGcloud.project
+        return UNSUPPORTED unless image.include? SamsonGcloud.project
+        return UNSUPPORTED unless image.include? "@" # only scanning shas is supported, not tags
```

Comment:
Instead of just returning just Unsupported would be better to return the error message why it is unsupported would be helpful.